SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp

```diff
@@ -208,6 +216,8 @@ WORKERCONNECTION_TEST(GIVEN_running_local_deployment_WHEN_connecting_client_and_
 	// WHEN
 	USpatialWorkerConnection* ClientConnection = NewObject<USpatialWorkerConnection>();
 	USpatialWorkerConnection* ServerConnection = NewObject<USpatialWorkerConnection>();
+	ClientConnection->AddToRoot();
```

Comment:
Quite a few duplicates of this

Reply:
Yes, the creation of server connection / client connection could be refactored into a function probably. Not sure if the cleanup of the connections should be as well (2 lines -> 1 line).